mercury-operator/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 quarkus.operator-sdk.crd.output-directory=../../crd
 application.version=${project.version}
-quarkus.operator-sdk.crd.validate=false
\ No newline at end of file
+quarkus.operator-sdk.crd.validate=false
+quarkus.kubernetes-client.namespace=mercury
```

Comment:
Why do you need to set the namespace? The operator should work on any namespace

Reply:
I did this because we said that the config map for the camel should be in the mercury namespace, but currently we are doing client.configMaps().inNamespace(client.getNamespace()).withName(sdConfigMapName).get(), If someone runs oc create -f sd.yaml in another namespace and not in the mercury namespace the operator wont find the config file for the integration

Reply:
There shouldn't be any reference to any namespace in the code. If you deploy the operator and all the dependent resources in a different namespace it should work.
The configMap is a dependency and should be deployed in the same namespace.

In the documentation you have to mention (if not done yet) that the configmaps must be created in the same namespace as the operator (as part of the operator deployment)